### PR TITLE
(2 min) Update CONTRIBUTING.md for yarn-link notes and linking to workflow investigation ticket

### PR DESCRIPTION
**Pull request by Cytancy**
Updated here: https://github.com/flexport/latitude/blob/ctan/update_contributing/CONTRIBUTING.md

### CODE REVIEW

**Comment by ddzoan** [comment]
I tried setting this up with bindfs and couldn't get the design system app to run locally. I'm getting `Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,35 +20,57 @@ This setup aims to reduce friction for internal Latitude contributors.
...
+cd location/of/latitude
+yarn link
+cd location/of/flexport
+yarn link latitude
+```
+
+*To unlink:*
+```
+cd location/of/flexport
+yarn unlink latitude
```
Comment (ShawnChenXHC) [inline CONTRIBUTING.md]: Add that you should run `yarn install --force` after this so that your `node_modules` folder are back to what they should be.

Comment (ShawnChenXHC) [reply CONTRIBUTING.md]: And also if they wish to remove `latitude` from the yarn link registry, they should run
```
cd location/of/latitude
yarn unlink
```

**Comment by Cytancy** [comment]
> I tried setting this up with bindfs and couldn't get the design system app to run locally. I'm getting `Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`

Confirmed, doesn't work for me either, tried the following: `CoreApp`, `DesignSystemApp`, `DispatchApp`, `ClientApp`

File: CONTRIBUTING.md (hunk 1 above)
Comment (Cytancy) [reply CONTRIBUTING.md]: Nice, thank you! Updated 👍

**Comment by Cytancy** [comment]
@jxu-flexport Did you happen to run into issues with webpack using `bindfs`?

**Comment by jxu-flexport** [comment]
@Cytancy `bindfs` was working for me with this change https://github.flexport.io/flexport/flexport/pull/56755

**Comment by Cytancy** [comment]
Ah I see, sorry for missing that, taking a look 👀

Thank you!

**Pull request merged**